dataline-api/src/main/openapi/config.yaml

```diff
@@ -927,6 +927,9 @@ components:
       properties:
         destinationImplementationId:
           $ref: "#/components/schemas/SourceImplementationId"
+    DestinationConfigurationField:
```

Comment:
I added this extra object to make the destination & source symmetrical. Let me know if this is wrong.